01_Data/src/layers/sequelize/model/TransactionEvent/Transaction.ts

```diff
@@ -60,30 +61,30 @@ export class Transaction extends Model implements OCPP2_0_1.TransactionType {
   declare totalKwh?: number | null;
 
   @Column(DataType.STRING)
-  declare stoppedReason?: OCPP2_0_1.ReasonEnumType | null;
+  declare stoppedReason?: string | null;
 
   @Column(DataType.INTEGER)
   declare remoteStartId?: number | null;
 
   @Column(DataType.DECIMAL)
   declare totalCost?: number;
-  
-  declare customData?: OCPP2_0_1.CustomDataType | null;
+
+  declare customData?: object | null;
```

Comment:
`object` might cause an error in the `TransactionMapper`. If so, we can change here to `any`